pkgs/sagemath-bliss/MANIFEST.in

```diff
@@ -1,12 +1,13 @@
-global-include all__sagemath_bliss.py
-
 include VERSION.txt
 
 graft sage/graphs/bliss_cpp
```

Comment:
What is this file? Is it supposed to be `bliss.cpp`, a generated file from `bliss.pyx`?

Reply:
It's a directory that contains `bliss_find_automorphisms.h` since a recent update

Reply:
OK, that is stuff from the bliss update, I thought that was merged in develop, that explains why I cannot see it.